.circleci/config.yml

```diff
@@ -1357,27 +1389,13 @@ workflows:
         - not: << pipeline.parameters.generate_snapshots >>
         - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
     jobs:
-      - lint:
-          context:
-            - slack-secrets-ios
-      - run-test-ios-17:
-          context:
-            - slack-secrets-ios
-      - run-test-ios-18:
-          context:
-            - slack-secrets-ios
-      - pod-lib-lint:
-          context:
-            - slack-secrets-ios
-      - run-revenuecat-ui-ios-17:
-          context:
-            - slack-secrets-ios
-      - run-revenuecat-ui-ios-18:
-          context:
-            - slack-secrets-ios
-      - emerge_purchases_ui_snapshot_tests:
-          context:
-            - slack-secrets-ios
+      - lint
+      - run-test-ios-17
+      - run-test-ios-18
+      - pod-lib-lint
+      - run-revenuecat-ui-ios-17
+      - run-revenuecat-ui-ios-18
+      - emerge_purchases_ui_snapshot_tests
```

Comment:
This is `build-test` and it runs on PRs so we don't want a notification in that case